src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -85,6 +85,30 @@ public void testComplexSelection()  throws IOException {
         spec.executeTest("testComplexSelection--" + testFile, this);
     }
 
+    @Test
+    public void testUntrimmedVariants() throws IOException {
```

Comment:
This test needs some javadoc explaining the potential output ordering pitfall of the input untrimmed variants.

Reply:
done